lib/fix_db_schema_conflicts/tasks/db.rake

```diff
@@ -13,7 +13,14 @@ namespace :db do
       end
       autocorrect_config = FixDBSchemaConflicts::AutocorrectConfiguration.load
       rubocop_yml = File.expand_path("../../../../#{autocorrect_config}", __FILE__)
-      `bundle exec rubocop --auto-correct --config #{rubocop_yml} #{filename.shellescape}`
+      `bundle exec rubocop --#{auto_correct_arg} --config #{rubocop_yml} #{filename.shellescape}`
+    end
+
+    private
```

Comment:
I think `private` is useless here as we're in rake namespace, not a class definition. 

Defining a function in a rake file also makes it a global:
```ruby
$ irb 
2.6.6 :001 > require 'rubocop'
2.6.6 :002 > require 'rake'
2.6.6 :003 > Dir.glob('**/*.rake').each { |p| load p }
 => ["lib/fix_db_schema_conflicts/tasks/db.rake"]

2.6.6 :004 > auto_correct_arg
 => "autocorrect"
# ^^^ can be just called from anywhere, may create fun effects in bigger projects
```


Let's either extract this into `FixDBSchemaConflicts::AutocorrectConfiguration` or similar new class nearby (which will also let us cover it with a spec, nice bonus) or just make it a variable in `dump` task:

```ruby
auto_correct_arg = if Gem.loaded_specs['rubocop'].version >= Gem::Version.new('1.30')
  'autocorrect'
else
  'auto-correct'
end
```

For such a small check I can accept it just existing like this without a spec.

Reply:
@bak1an I have made the change you requested

Reply:
Looks good, thanks!